README.md

```diff
@@ -33,7 +33,7 @@ All operations between the JavaScript application code and the native platform a
 
 ## Touch Handling
 
-iOS has a very powerful system called the Responder Chain to negotiate touches in complex view hierarchies which does not have a universal analog on the web. React Native implements a similar responder system and provides high level components such as TouchableHighlight that integrate properly with scroll views and other elements without any additional configuration.
+iOS has a very powerful system called the Responder Chain to negotiate touches in complex view hierarchies which do not have a universal analog on the web. React Native implements a similar responder system and provides high level components such as `TouchableHighlight`, which integrate properly with scroll views and other elements without any additional configuration.
```

Comment:
this change is incorrect - it is the responder chain that *does* not have a universal analog